docker/micro-cluster-lab/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:bionic
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-amd64/jre
```

Comment:
usage of this syntax (without `=` sign) [is discouraged](https://docs.docker.com/engine/reference/builder/#env). I suggest that we use newer `ENV <key>=<value>` syntax across this file

Reply:
Fixed